src/plugins/intel_gna/tests/functional/shared_tests_instances/behavior/ov_plugin/core_integration.cpp

```diff
@@ -151,9 +151,6 @@ TEST(OVClassBasicTest, smoke_SetConfigAfterCreatedPerformanceHint) {
     ov::Core core;
     ov::hint::PerformanceMode mode;
 
-    OV_ASSERT_NO_THROW(mode = core.get_property("GNA", ov::hint::performance_mode));
-    ASSERT_EQ(ov::hint::PerformanceMode::UNDEFINED, mode);
-
```

Comment:
not sure if this is covered in other tests, but probably to be consistent it should be ASSERT_EQ(ov::hint::PerformanceMode::LATENCY, mode);